### ISSUE TITLE
Question: does a sniff exist to enforce this way of writing a ternary statement?

### ISSUE THREAD

**Issue by u01jmg3** [squizlabs/PHP_CodeSniffer#2610]
```php
// Disallow
(empty($string)) ? $string .= 'Blank' : $string .= 'Filled';

// Allow
$string .= (empty($string)) ? 'Blank' : 'Filled';
```

**Comment by gsherwood** [comment]
PHPCS does not come with a sniff to enforce this sort of rule.

It's possible that someone else has written one for another project's standard, but I'm not personally aware of one.

**Comment by u01jmg3** [comment]
Hopefully somebody else has addressed this before 🤞
